src/main/res/values/strings.xml

```diff
@@ -120,47 +120,47 @@
   <string name="saving_details">Saving details</string>
   <string name="screenshots">Screenshots</string>
   <string name="search">Search</string>
-  <string name="select_the_mirror">Select the mirror</string>
-  <string name="show_more">Show more</string>
-  <string name="show_older_releases">Show older releases</string>
+  <string name="select_the_mirror">Select a mirror</string>
+  <string name="show_more">More</string>
+  <string name="show_older_releases">Show older versions</string>
```

Comment:
"Show older releases" is more suitable, since category is named "Releases".

Reply:
version/s is used elsewhere, and "releases" I thought could be avoided.

Reply:
"Releases" string should be changed to "Versions", then. But I still prefer "Release" word :/